tests/tools/importexport/migration/test_v08_to_v09.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Test export file migration from export version 0.8 to 0.9"""
+from aiida.backends.testbase import AiidaTestCase
+from aiida.tools.importexport.migration.utils import verify_metadata_version
+from aiida.tools.importexport.migration.v08_to_v09 import migrate_v8_to_v9
+
+from tests.utils.archives import get_json_files
+
+
+class TestMigrate(AiidaTestCase):
```

Comment:
```suggestion
class TestMigrateV08toV09(AiidaTestCase):
```
Not crucial - simply to keep in step with the other Class names.

Reply:
I agree the inconsistency with the existing test files is not nice, but I decided to do this on purpose to simplify creating these test files. Each file contains a lot of repetition of the two versions concerned making for a lot of work to make sure all occurrences are properly replaced. The two changes you have highlighted are not the only occurrences. Do you prefer I revert those changes or I simplify all other test files as well to remove references to version numbers. As I see it, since the module name itself already includes the versions, that is more than enough

Reply:
I vote for simplification in the name of hindsight 🗳️